server/xpub/entities/manuscript/resolvers.js

```diff
@@ -43,73 +43,68 @@ const resolvers = {
 
       return manuscripts[0]
     },
-    async manuscript(_, { id }) {
-      return Manuscript.find(id)
+    async manuscript(_, { id }, { user }) {
+      return Manuscript.find(id, user)
     },
-    async manuscripts() {
-      return Manuscript.all()
+    async manuscripts(_, vars, { user }) {
+      return Manuscript.all(user)
     },
   },
 
   Mutation: {
     async createSubmission(_, vars, ctx) {
+      if (!ctx.user) {
+        throw new Error('Not logged in')
```

Comment:
Log something too?

Reply:
Errors thrown in resolvers are logged to the console as well as being returned in the response.

Reply:
I remember having some issues with this in the past when I first started logging stuff, but can't remember exactly what.